packages/rn-tester/js/examples/TextInput/TextInputExample.ios.js

```diff
@@ -611,6 +611,12 @@ exports.examples = ([
             multiline={true}
             style={styles.multiline}
           />
+          <TextInput
+            placeholder="multiline text input with scroll disabled"
+            multiline={true}
+            scrollEnabled={false}
+            style={styles.multiline}
+          />
```

Comment:
Also macOS tags here.

Reply:
@shwanton seems like this could be upstreamed to RN core if you want.